namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-manage-adjudications-api-dev/resources/rds.tf

```diff
@@ -15,10 +15,10 @@ module "ma_rds" {
   namespace                   = var.namespace
   environment-name            = var.environment-name
   infrastructure-support      = var.infrastructure-support
-  allow_major_version_upgrade = "false"
+  allow_major_version_upgrade = "true"
   db_instance_class           = "db.t3.small"
-  db_engine_version           = "10"
```

Comment:
The instance is currently running `10.21` and checking on [Upgrading the PostgreSQL DB engine for Amazon RDS
](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_UpgradeDBInstance.PostgreSQL.html#USER_UpgradeDBInstance.PostgreSQL.MajorVersion) this can be upgraded to `14.3`